### DISCUSSION TITLE
(Beginner) How to load arabert model (tf_model.h5) to keras layer?

### DISCUSSION THREAD

**Discussion by faiz-hub** [aub-mind/arabert#85] [Q&A] [answered]
Dear NLP Expert,
Any guidance on how to load_model or load_weights of downloadable arabert model (eg. tf_model.h5) to keras layer?,

Thank you in advance,
Faiz

**Comment by WissamAntoun** [comment, marked as answer]
check the answer here if it helps https://stackoverflow.com/questions/63201036/add-additional-layers-to-the-huggingface-transformers

**Reply by faiz-hub** [reply]
Sure it helped
Thank you wissam
جزاك الله خيرا
